config/routes.rb

```diff
@@ -126,6 +126,23 @@
       to: "users#index",
       as: :users
 
+  resources :users,
+            only: [],
+            path: "/bo/users",
+            path_names: { show: "/:id" } do
```

Comment:
Sorry just spotted this! Shall we get rid of this Path name as well? I am pretty sure is not used